### ISSUE TITLE
transformed_dataset And dataset_similarity

### ISSUE THREAD

**Issue by NookLook2014** [Merlin55/PTARL#1]
Could you please sharing the data set mentioned in your code file
path_data = '/home/Data/Pengyang/T-drive-taxi/transformed_dataset'
path_similarity = '/home/Data/Pengyang/T-drive-taxi/dataset_similarity'
ppath = '/home/Data/Pengyang/T-drive-taxi/train_dataset'

s sample or a description is also welcome!

**Comment by hichamatr** [comment]
I think that they used this :
https://www.microsoft.com/en-us/research/publication/t-drive-trajectory-data-sample/?from=http%3A%2F%2Fresearch.microsoft.com%2Fapps%2Fpubs%2F%3Fid%3D152883

**Comment by NookLook2014** [comment]
th

> I think that they used this :
> https://www.microsoft.com/en-us/research/publication/t-drive-trajectory-data-sample/?from=http%3A%2F%2Fresearch.microsoft.com%2Fapps%2Fpubs%2F%3Fid%3D152883

this link is only the raw data, i need the transformed data after data preprocess.

**Comment by DenisDPR** [comment]
@NookLook2014 did you get this data. Please @Merlin55 could you share the description of this datasets (Not the raw data)

**Comment by NookLook2014** [comment]
Nope, the author didn't give any response though mail sent. And this paper has been published in TKDE, but i really doubt about the quality of this datasets....




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Denis Pastory"<notifications@github.com&gt;; 
发送时间: 2020年3月4日(星期三) 凌晨0:22
收件人: "Merlin55/PTARL"<PTARL@noreply.github.com&gt;; 
抄送: "卧水之冰"<383302086@qq.com&gt;; "Mention"<mention@noreply.github.com&gt;; 
主题: Re: [Merlin55/PTARL] transformed_dataset And dataset_similarity (#1)




@NookLook2014 did you get this data. Please @Merlin55 could you share the description of this datasets (Not the raw data)
